packages/eslint-plugin/src/rules/no-invalid-void-type.ts

```diff
@@ -31,6 +32,8 @@ export default util.createRule<[Options], MessageIds>({
         'void is only valid as return type or type of `this` parameter.',
       invalidVoidNotReturnOrThisParamOrGeneric:
         'void is only valid as a return type or generic type variable or the type of a `this` parameter.',
+      InvalidVoidForUnion:
+        'Void is valid when any types used together only as part of a union type.',
```

Comment:
This sentence doesn't make sense to me. Also, `void` is lowercase in the other messages. How about...

```suggestion
        'void is not valid as a type constituent in a return type.',
```

Reply:
ok! I fixed this error message!